scripts/actions/mobskills/stupor_spores.lua

```diff
@@ -0,0 +1,26 @@
+-----------------------------------
+-- Stupor Spores
+-- Family: Euvhi
+-- Description: Spreads intoxicating spores that damages nearby targets. Additional effect: Sleep
+-- Type: Magical
+-- Utsusemi/Blink absorb: Ignores shadows
+-----------------------------------
+---@type TMobSkill
+local mobskillObject = {}
+
+mobskillObject.onMobSkillCheck = function(target, mob, skill)
+    return 0
+end
+
+mobskillObject.onMobWeaponSkill = function(target, mob, skill)
+    local damage = xi.mobskills.mobMagicalMove(mob, target, skill, mob:getMainLvl() + 2, xi.element.NONE, 1, xi.mobskills.magicalTpBonus.NO_EFFECT)
```

Comment:
Jimmy's sheet says this is `1 + dINT`, is this right?

Reply:
I think I'm a bit confused on how to implement that then. I saw that dINT is being calculated within mobMagicalMove so I wasn't sure if I should do anything on the skill itself.